src/prefect/cli/deployment.py

```diff
@@ -768,3 +851,62 @@ async def build(
                 "edit the deployment spec and re-run this command, or visit the deployment in the UI.",
                 style="red",
             )
+
+
+def _load_json_key_values(
+    cli_input: List[str], display_name: str
```

Comment:
This display name existed in v1 to support `context` and `parameter` parsing with a single function. We should probably drop it here.